README.md

```diff
@@ -97,6 +97,15 @@ Also remove some shaders mods:
 * 190- Screen Space Shaders - Ascii1457
 * 290- Atmospherics Shaders Weathers and Reshade - Hippobot
 
+### ModuleNotFoundError: No module named 'distutils'
+
+The `distutils` module is required to install Python packages but it was removed in Python 3.12. You can still use it by installing `setuptools` (inside the venv): `pip install setuptools`
+
+### rarfile.RarCannotExec: Cannot find working tool
+
+You are missing a tool that extracts RAR files. You can use something like `unrar` on Linux. To install it run the following command:
+- On Ubuntu: `sudo apt install unrar`
+- On Fedora: `sudo dnf install unrar`
```

Comment:
Please precise the need of using RPM fusion, according to [pkgs.org](https://fedora.pkgs.org/39/fedora-x86_64/unrar-0.1.3-2.fc39.x86_64.rpm.html), default **unrar** package is a meta-package which will install **unrar-free** and this will cause problems. (**unrar-free** is not supported by rarfile)

Packages to use:
https://fedora.pkgs.org/39/rpmfusion-nonfree-x86_64/unrar-6.2.12-1.fc39.x86_64.rpm.html

Reply:
Thanks for noticing that! It is added now.